### PR TITLE
Qiskit in galaxy

### PR DESCRIPTION
**Pull request by thepineapplepirate**
This PR is for the XML file for a new interactive tool, "Qiskit in Galaxy". This tool uses a docker image that contains packages for GPU-enabled jupyter lab, as well as all relevant packages, resources, and tutorials for Qiskit. This tool will allow Galaxy users to learn and leverage quantum computing, and submit jobs to IBM quantum hardware from a Galaxy instance and import those results back into the history. Users will need to pass their IBM credentials (API token) under User Preferences in the Galaxy instance in order to use actual quantum hardware in the backend. Alternatively, users can readily work with quantum simulators (comes with the Qiskit modules) locally wherever the galaxy instance is, without needing IBM credentials. 

All files associated with the docker image can be found here:
https://github.com/thepineapplepirate/qiskit-jupyter-galaxy_docker.git 

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [ ] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

### CODE REVIEW

File: tools/interactive/interactivetool_qiskit_jupyter_notebook.xml [hunk 1]
```diff
@@ -0,0 +1,314 @@
+<tool id="interactive_tool_qiskit_jupyter_notebook" tool_type="interactive" name="Qiskit Jupyter notebook" version="0.0.1" profile="22.01">
+    <description>interactive tool</description>
+    <requirements>
+        <container type="docker">thepineapplepirate/qiskit_galaxy:1.0.0</container>
+     	## files for this docker image can be found at https://github.com/thepineapplepirate/qiskit-jupyter-galaxy_docker.git ##
```
Comment (bgruening) [inline tools/interactive/interactivetool_qiskit_jupyter_notebook.xml]: this is not a proper XML comment, does this work actually?

Comment (thepineapplepirate) [reply tools/interactive/interactivetool_qiskit_jupyter_notebook.xml]: I was worried that this wasn't the proper way. It does actually work though as is, I have it running locally. Please let me know how I should change it and will go ahead and do so :).

Comment (nsoranzo) [reply tools/interactive/interactivetool_qiskit_jupyter_notebook.xml]:
```suggestion
     	<!-- files for this docker image can be found at https://github.com/thepineapplepirate/qiskit-jupyter-galaxy_docker.git -->
```

See https://www.tutorialspoint.com/xml/xml_comments.htm

**Comment by thepineapplepirate** [comment]
Thanks @nsoranzo and @bgruening for reviewing!

**Review by bgruening — APPROVED** [review]
Thanks @thepineapplepirate!

**Pull request merged**